functions_assignment.js

```diff
@@ -0,0 +1,240 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+function largerChoice (num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 >= num1) {
+    return num2;
+  }
+}
+console.log(largerChoice(12,4));
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument:
+// a dog's age in human years. The function returns the dog's age in dog years.
+// The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+const getDogAge = (num) => {
+  return num * 7;
+}
+console.log(getDogAge(4));
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the sum of all
+// the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+let myArray = [13, 2, 4, -2];
+const arraySum = (arr) => {
+  let sum = 0;
+  for (i in arr) {
+    sum += arr[i];
+    arr[i]++;
+  }
+  return sum;
+}
+console.log(arraySum(myArray));
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+function sumTwoNums (num1, num2) {
+  return num1 + num2;
+}
+console.log(sumTwoNums(14,5));
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+const letterGrade = (num1) => {
+    if (num1 === 100) {
+      return "A+";
+    } else if (num1 >= 90 && num1 <= 99) {
+      return "A";
+    } else if (num1 >= 80 && num1 <= 89) {
+      return "B";
+    } else if (num1 >= 70 && num1 <= 79) {
+    return "C";
+    } else if (num1 >= 65 && num1 <= 69) {
+    return "D";
+    } else {
+    return "F";
+  }
+}
```

Comment:
Great. With knowledge on how code control flows, how can you make your code simpler?